cache.go

```diff
@@ -13,6 +13,8 @@ import (
 const (
 	NoExpiration      time.Duration = -1 // Never expires
 	DefaultExpiration time.Duration = 0  // Use default config value
+
+	CachedTimeStampUpdaterIntervals = 100 * time.Millisecond // How often should updater run
```

Comment:
Do you need this variable, and other to be exported ?

I mean, do you consider people would use `kioshun.CachedTimeStampUpdaterIntervals = delay` ?